### ISSUE TITLE
how to create a .jsh file on windows?

### ISSUE THREAD

**Issue by DerrickSaltFish** [nus-cs2030/2122-s1#42]
Can I create a .txt file and simply rename it as .jsh?

**Comment by RussellDash332** [comment]
You might not change the extension of the file, rather you're adding the `.jsh` to the filename so that it's now `<filename>.jsh.txt`.
Why not adding it using the command line instead?
- Use vim to create an empty file and quit it right away. I.e. `vim <filename>.jsh` and then `:q`
- Use `touch <filename>.jsh`

**Comment by jeremiahlzz** [comment]
If u really want to, you can:
 - Create a .txt file
 - Open it
 - On the top left, under file, save as
 - For filename, use "<filename>.jsh"
 - Make sure to **change Save as type** to "All Files"
 - You should now have a JSH filetype

**Comment by francisyzy** [comment]
> 
>     * Use `touch <filename>.jsh`

Is not a windows command thing unless he uses WSL. 

Here is a `blank.jsh` in a zip file if you wanna download an empty `.jsh` for ur use
[blank.jsh.zip](https://github.com/nus-cs2030/2122-s1/files/7112193/blank.jsh.zip)

**Comment by RussellDash332** [comment]
> > ```
> > * Use `touch <filename>.jsh`
> > ```
> 
> Is not a windows command thing unless he uses WSL.
> 
> Here is a `blank.jsh` in a zip file if you wanna download an empty `.jsh` for ur use
> [blank.jsh.zip](https://github.com/nus-cs2030/2122-s1/files/7112193/blank.jsh.zip)

Yes, I just realized that, my bad 😄 
I also just realized `.jsh` is an option while saving a `.txt` file in Notepad, thanks @jeremiahlzz !

**Comment by lhy-hoyin** [comment]
> Can I create a .txt file and simply rename it as .jsh?

Yes you can. In Windows Explorer, ensure the "File name extensions" is checked under the "View" tab.
![image](https://user-images.githubusercontent.com/88828097/132153370-4bb9933a-d847-46eb-9624-b882f4b9943e.png)

Then right click on the file and click rename (or simply press `F2`). Go ahead and rename the extension of the file to `.jsh`.

**Comment by Yuxinn-J** [comment]
`touch <filname>.jsh` to create
then
`vim <filename>.jsh` to edit

Hope this helps 😄
